src/browser/setupWorker/start/createRequestListener.ts

```diff
@@ -47,18 +48,29 @@ export const createRequestListener = (
             // ".log()" method of the request handler.
             const responseClone = response.clone()
             const responseInit = toResponseInit(response)
-            const responseStream = responseClone.body
 
-            messageChannel.postMessage(
-              'MOCK_RESPONSE',
-              {
+            /**
+             * @note Safari doesn't support transferring a "ReadableStream".
+             * Check that the browser supports that before sending it to the worker.
+             */
+            if (supportsReadableStreamTransfer()) {
```

Comment:
I dislike checking this on _every_ response listener. 

This should be determined _once_ because its value is bound to the current browser and cannot change on runtime. 

We should move this somewhere to the `setupWorker()` call, I think. It's computed once, even before handling the first response, and stored in the worker context. Something like `worker.context.supportsReadableStreamTransfer` can work.